js/components/portal-dashboard/feedback/rubric-table.tsx

```diff
@@ -0,0 +1,155 @@
+  updateActivityFeedback?: (activityId: string, activityIndex: number, platformStudentId: string, feedback: any) => void;
+}
+export class RubricTableContainer extends React.PureComponent<IProps> {
+  render() {
+    const { activityStarted, rubric } = this.props;
+    const { criteria } = rubric;
+
+    if (activityStarted) {
+      return (
+        <div className={css.rubricContainer} data-cy="rubric-table">
+          {this.renderColumnHeaders(rubric)}
+          <div className={css.rubricTable}>
+            {criteria.map((crit: any) =>
+              <div className={css.rubricTableRow} key={crit.id} id={crit.id}>
+                <div className={css.rubricDescription}>
+                  <Markdown>{crit.description}</Markdown>
+                </div>
+                {this.renderRatings(crit)}
+              </div>
+            )}
+          </div>
+        </div>
+      );
+    }
+
+    return null;
+  }
+
+  private renderColumnHeaders = (rubric: any) => {
+    const { referenceURL } = rubric;
+    return (
+      <div className={css.columnHeaders}>
+        <div className={css.rubricDescriptionHeader}>
+          <div className={css.scoringGuideArea}>
+            <a className={css.launchButton}>
+              <LaunchIcon className={css.icon} href={referenceURL} target="_blank" />
+            </a>
+            Scoring Guide
+        </div>
+          <div className={css.rubricDescriptionTitle}>{rubric.criteriaLabel}</div>
+        </div>
+        { rubric.ratings.map((rating: any) =>
+          <div className={css.rubricScoreHeader} key={rating.id}>
+            <div className={css.rubricScoreLevel}>{rating.label}</div>
+            {rubric.scoreUsingPoints && <div className={css.rubricScoreNumber}>({rating.score})</div>}
+          </div>
+        )
+        }
+      </div>
+    );
+  }
+
+  private renderRatings = (crit: any) => {
+    const { rubric } = this.props;
+    const { ratings } = rubric;
+    return (
+      <div id={`${crit.id}_ratings`} className={css.ratingsGroup}>
+        {ratings.map((rating: any, index: number) => this.renderStudentRating(crit, rating, index))}
+      </div>
+    );
+  }
+
+  private renderStudentRating = (crit: any, rating: any, buttonIndex: number) => {
+    const { rubricFeedback } = this.props;
+
+    const critId = crit.id;
+    const ratingId = rating.id;
+    const radioButtonKey = `${critId}-${ratingId}`;
+    const selected = (rubricFeedback && rubricFeedback[critId] && rubricFeedback[critId].id === ratingId);
+    // Tooltips displayed to teacher should actually show student description if it's available.
+    const ratingDescription =
+      (crit.ratingDescriptionsForStudent && crit.ratingDescriptionsForStudent[ratingId]) ||
+      (crit.ratingDescriptions && crit.ratingDescriptions[ratingId]) ||
+      null;
+    const isApplicableRating = crit.nonApplicableRatings === undefined ||
+      crit.nonApplicableRatings.indexOf(ratingId) < 0;
+    const isNotApplicable = !isApplicableRating;
```

Comment:
You could eliminate the local variable by simply using either `isApplicableRating` or `!isApplicableRating` in the test below.